### ISSUE TITLE
Travis Python 2.7 tests stall on installing dependencies

### ISSUE THREAD

**Issue by lilyminium** [MDAnalysis/mdanalysis#3098]
## Expected behavior ##

Tests pass

## Actual behavior ##

The python 2.7 test consistently stalls upon installing dependencies. See https://travis-ci.com/github/MDAnalysis/mdanalysis/jobs/472541183#L641

## Code to reproduce the behavior ##

``` python
import MDAnalysis as mda
from MDAnalysis.tests.datafiles import PSF, DCD,  GRO, PDB, TPR, XTC, TRR,  PRMncdf, NCDF

u = mda.Universe(PSF, DCD)

....

```

## Current version of MDAnalysis ##

- Which version are you using? (run `python -c "import MDAnalysis as mda; print(mda.__version__)"`)
- Which version of Python (`python -V`)? master
- Which operating system?

**Comment by orbeckst** [comment]
Duplicate of #3079 ?

**Comment by lilyminium** [comment]
@orbeckst Oops yes, I have a short memory....

Closed by #3095 and #3097.
